### ISSUE TITLE
importing ontologies without aeon:SMW_datatype 

### ISSUE THREAD

**Issue by andrecastro0o** [TIBHannover/ontology2smw#31]
Currently ontology2smw import to wiki only the terms which contain a value to the property `aeon:SMW_datatype` as visible in SPARQL [query_classes_properties.rq](https://github.com/TIBHannover/ontology2smw/blob/master/ontology2smw/queries/query_classes_properties.rq)

I would like to make ontology2smw able to import other ontologies terms, even if they do not have `aeon:SMW_datatype`, as for instance [datacite](https://sparontologies.github.io/datacite/current/datacite.html) [datacite ttl](https://sparontologies.github.io/datacite/current/datacite.ttl) OR GND Ontology ([gndo](https://d-nb.info/standards/elementset/gnd)) [gndo ttl](https://d-nb.info/standards/elementset/gnd.ttl) 

Development Steps
- [x] SPARQL query: to extract information from ontology terms
- [x] Test ontology query
- [x] Mapping xsd datatypes to SMW datatypes
- [x] All namespaces & prefix dict, derived from http://prefix.cc/
- [x] Prompt user for prefix when not found in all prefixes dict
- [x] Implement

**Comment by andrecastro0o** [comment]
SPARQL Query in https://github.com/TIBHannover/ontology2smw/blob/issue31_ontologies_wout_aeon/ontology2smw/queries/ontology_terms.rq commit: https://github.com/TIBHannover/ontology2smw/commit/55d87fd414ef763a825c603d50a12d80dd3cbca9

**Comment by andrecastro0o** [comment]
# Find the datatypes

## is rdf:type value always a owl:XYZ ?
Or do ontology designers use other strategies to define a term as class / DatatypeProperty / ObjectProperty?

## determine `Has type::` without aeon:SMW_datatype
The main issue is how to determine from the terms definition the SMW `Has type::` property of a Property definition

In ontology schema `rdf:type` is use to define if a terms is:
* `owl:Class` -> SMW: Category
* `owl:AnnotationProperty` -> SMW: `Has type::Text`
* `owl:ObjectProperty` -> SMW: `Has type::Page`
* `owl:DatatypeProperty`  -> SMW: `Has type:: ???`


https://www.w3.org/TR/2004/REC-owl-guide-20040210/#owl_DatatypeProperty show the recommended xsd datatypes for use with `owl:DatatypeProperty`, plus `rdfs:Literal`
 
### where is datatype defined?

**In rdfs:range**: the range of values given to a DatatypeProperty are X

```ttl
###  https://github.com/tibonto/aeon#ID_URL
aeon:ID_URL rdf:type owl:DatatypeProperty ;
            rdfs:subPropertyOf aeon:ID ;
            rdfs:domain aeon:Identifier ;
            rdfs:range xsd:anyURI ;
```
```ttl
###  http://rs.tdwg.org/abcd/terms/acquisitionSourceText
:acquisitionSourceText rdf:type owl:DatatypeProperty ,
                                owl:FunctionalProperty ;
                       rdfs:domain owl:Thing ;
                       rdfs:range xsd:string ;
```

```ttl
dcat:bbox
  a rdf:Property ;
  a owl:DatatypeProperty ;
  rdfs:domain dct:Location ;
  rdfs:label "bounding box"@en ;
  rdfs:range rdfs:Literal ;
```
```ttl
dcat:spatialResolutionInMeters
  a owl:DatatypeProperty ;
  rdfs:range xsd:decimal ;
```

```ttl 
###  http://xmlns.com/foaf/0.1/age
:age rdf:type owl:DatatypeProperty ,
              owl:FunctionalProperty ;
     rdfs:domain :Agent ;
     rdfs:range rdfs:Literal ;
```

**Comment by andrecastro0o** [comment]
## Mapping  xsd datatypes  to [SMW datatypes](https://www.semantic-mediawiki.org/wiki/Help:List_of_datatypes)

```python
{
'xsd:string': 'Text',  
'rdfs:Literal': 'Text',  
'xsd:Name': 'Text',  
'xsd:normalizedString': 'Text',  
'xsd:decimal': 'Number',  
'xsd:float': 'Number',  
'xsd:integer': 'Number',  
'xsd:nonNegativeInteger': 'Number',  
'xsd:positiveInteger': 'Number',  
'xsd:nonPositiveInteger': 'Number',  
'xsd:negativeInteger': 'Number',  
'xsd:positiveInteger': 'Number',  
'xsd:int': 'Number',  
'xsd:double': 'Number',  
'xsd:long': 'Number',  
'xsd:short': 'Number',  
'xsd:unsignedLong': 'Number',  
'xsd:byte': 'Number',  
'xsd:boolean': 'Boolean',  
'xsd:dateTime': 'Date',  
'xsd:time': 'Text',  
'xsd:date': 'Date',  
'xsd:gYearMonth': 'Date',  
'xsd:dateTime': 'Date',  
'xsd:gYear': 'Date',  
'xsd:gMonthDay': 'Text',  
'xsd:gDay': 'Text',  
'xsd:gMonth': 'Text',  
'xsd:anyURI': 'URL',  
'xsd:language': 'Text'  
}
```

**Comment by andrecastro0o** [comment]
Prefixes & Namespaces from http://prefix.cc/ in [ontology2smw/queries/all_ns_prefixes.json](https://github.com/TIBHannover/ontology2smw/blob/issue31_ontologies_wout_aeon/ontology2smw/queries/all_ns_prefixes.json)

**Comment by andrecastro0o** [comment]
closed by #56
